src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Queries/ContentItemsFieldType.cs

```diff
@@ -77,12 +77,8 @@ private async Task<IEnumerable<ContentItem>> Resolve(IResolveFieldContext contex
             JObject where = null;
             if (context.HasArgument("where"))
             {
-                var whereArgument = context.Arguments["where"];
-
-                if (whereArgument.Value != null)
-                {
-                    where = JObject.FromObject(whereArgument.Value);
-                }
+                // context.Arguments[].Value is never null in GraphQL.NET 4
```

Comment:
it can not have a value though i presume ?

Reply:
its a dictionary, it should always have a value, i checked for real data inside and it looks ok to me.